libraries/api/src/middleware/auth.js

```diff
@@ -253,12 +253,11 @@ const remoteAuthentication = ({signatureValidator, entry}) => {
         // If authentication failed
         if (result.status === 'auth-failed') {
           res.set('www-authenticate', 'hawk');
-          const err = new Error('Authentication failed'); // This way instead of subclassing due to babel/babel#3083
-          err.name = 'AuthenticationError';
-          err.code = 'AuthenticationError';
-          err.message = result.message;
-          err.details = result;
-          throw err;
+          throw new ErrorReply({
+            code: 'AuthenticationFailed',
+            message: result.message,
+            details: result,
```

Comment:
So that `result` object - does it have the information about which scopes are missing anywhere in it? Should we be showing that information at all in the authorization case?

Reply:
This error is for bad Hawk credentials, not bad scopes - so no, not in this case.